.travis.yml

```diff
@@ -46,8 +46,13 @@ jobs:
         - export GO111MODULE=on
       script:
         - go mod vendor
-        - make test PARALLEL_COUNT=2
+        - kubectl --insecure-skip-tls-verify config current-context
+        - export TESTING_CONTEXT=`kubectl --insecure-skip-tls-verify config current-context`
+        - ./build/prepare-sa-test.sh
+        - RUN_SA_OPERATOR=TRUE make test PARALLEL_COUNT=2
```

Comment:
We can also save time a bit and call ./build/run-tests.sh directly as make test compile whole operator code which is not needed at this step. Compile was done on unit-test step already

Reply:
I would keep the two stages independent, compile should work in incremental way and just do nothing on the second call